packages/manager/modules/telecom-universe-components/src/telecom/pack/migration/pack-migration-process.service.js

```diff
@@ -161,6 +161,13 @@ export default /* @ngInject */ function($q, OvhApiPackXdsl, Poller) {
       });
     }
 
+    // Set contact phone if is set
+    if (migrationProcess.contactPhone) {
+      assign(postParams, {
+        contactPhone: migrationProcess.contactPhone,
+      });
```

Comment:
```suggestion
      Object.assign(postParams, {
        contactPhone: migrationProcess.contactPhone,
      });
```
lodash doesn't seem necessary here